iothub/service/tests/JobClient/JobClientTests.cs

```diff
@@ -71,15 +69,13 @@ private void TestVerify(JobResponse actualJobResponse)
         public void DisposeTest()
         {
             httpClientHelperMock.Setup(restOp => restOp.Dispose());
-            jobClient.Dispose();
             httpClientHelperMock.Verify(restOp => restOp.Dispose(), Times.Once());
         }
 
         [TestMethod]
-        public async Task CloseAsyncTest()
+        public void CloseAsyncTest()
         {
             httpClientHelperMock.Setup(restOp => restOp.Dispose());
-            await jobClient.CloseAsync().ConfigureAwait(false);
```

Comment:
Why remove the call to `CloseAsync()` in a test called `CloseAsyncTest()`?

Reply:
IOW, either add it back or remove the test if you assert the test is not needed.